examples/hitl/rearrange_v2/config/rearrange_v2.yaml

```diff
@@ -43,6 +43,6 @@ habitat_hitl:
     - agent_index: 1
       lin_speed: 10.0
       ang_speed: 300
-  hide_humanoid_in_gui: True
+  hide_humanoid_in_gui: False
```

Comment:
What does this do? Why flip it?

Reply:
This removes the humans from the GUI application entirely. This is a simple way of handling human visibility for first-person single-user applications.

This doesn't scale to multiplayer applications because _all_ humans are removed. Now, we have fine-grained control over  object visibility per-viewport. See https://github.com/facebookresearch/habitat-lab/pull/1959.